src/notation/qml/MuseScore/NotationScene/internal/ElementPopup.qml

```diff
@@ -0,0 +1,34 @@
+import QtQuick 2.15
+import QtQuick.Layouts 1.15
+import QtQuick.Controls 2.15
+import QtQuick.Controls 1.3
+
+import MuseScore.NotationScene 1.0
+import MuseScore.UiComponents 1.0
+import MuseScore.Ui 1.0
+import MuseScore.Inspector 1.0
+
+
```

Comment:
Please don't over-use newlines. Use at most one in a row, and don't put newlines before closing braces (see bottom of this file).